src/styles/theme.ts

```diff
@@ -3,6 +3,7 @@ export const theme = {
   fontFamily: 'Roboto',
   fontWhite: '#e2dace',
   background: '#13120e',
+  background2: '#204C39',
```

Comment:
utilizar nome descritivo ou mudar para key colors e descrever a cor
```suggestion
  cardBackground: '#204C39',
```

Reply:
Done